src/nodes/pause_button.gd

```diff
@@ -0,0 +1,37 @@
+extends TextureButton
+
+var game_node: Node
+var pause_menu: Control
+var pausable_group: Node
+var game: Game
+
+func _ready():
+	game_node = get_tree().current_scene
+	
+	pausable_group = game_node.get_node("Pausable")
+	pause_menu = game_node.get_node("Stickable/PauseMenu")
+	game = SceneSwitcher.get_game_state()
+	
+	self.connect("pressed", _on_game_paused)
```

Comment:
I think you could have used the signals presented in the node section, but this is fine
![image](https://github.com/rutesantos4/ram-killer-scale-game-off-2023/assets/104737871/689616fa-897f-4cf4-a9fc-2c6bfcd4982c)

Reply:
I completely forgot about that, thank you!